### ISSUE TITLE
MMS not working at all

### ISSUE THREAD

**Issue by GithubGrisen** [ubports/ubports-touch#276]
- Device: Bacon
- Channel: Stable
- Build: 15.04 OTA2 - build number ? (Where do I find that?)

## Steps to reproduce

1. ... Send MMS
2. ... Recieve MMS
3. ...

## Expected behavoir
MMS's should be sent and recieved
...

## Actual behavior
Sending a picture from the messaging app gets a red circle to the left saying Fejl (Danish for error)
I don't recieve any MMS's at all
...

## Logfiles and additional information
I have checked th APN-settings, and they are correct. I'm using the danish operator TDC. 
I can attach logfiles upon request, but which files are necessary?
...

**Comment by mihaelmilea** [comment]
I don't think it ever worked for me... turbo, devel, r36.

**Comment by Flohack74** [comment]
Neither does it work on E5 - well thats strange, people reported it worked for them. We need to take a look at this soon.

**Comment by ernesst** [comment]
Fyi on N5, latest devel MMS works fine

**Comment by alexanderisme** [comment]
On Bq 4.5 with the canonical image it works... Have you the mobile data turned on ?

**Comment by Flohack74** [comment]
Yes mobile data is always on. I will try on N5 too. Can it be that the provider parameters are not detected automatically (from SIM card)? I remember that providers are sending out "configuration SMS" to get this done. Probably UT cannot understand them.

**Comment by mihaelmilea** [comment]
I have mine configured manually with the parameters from the provider's website and it's still not working...

**Comment by tomoqv** [comment]
The MMS functionality on Ubuntu Touch seems to depend on telecom operator. I recently switched from Tele2 (Sweden) to Telenor and now MMS works on my OPO.

**Comment by Flohack74** [comment]
Does anyone of you uses 2 SIM cards? This seems to make the problem more complicated. My GF is using T-Mobile with one SIM-card, and I have T-Mobile, too, but with 2 cards. She can send MMS (Did not see receiving), but I cant neither.

**Comment by Flohack74** [comment]
Guys can we please retest this with some 16.04 devices? my OPO is unfortunately on 15.04 as daily.

**Comment by tomoqv** [comment]
It seems not to work on my Pro 5 with Comviq. I will try with Telenor as well as soon as I have the time.

**Comment by GithubGrisen** [comment]
I have an OPO on 16.04, but it's been a while since I used it. I just updated it, but is that enough?

**Comment by Flohack74** [comment]
Yes, you should see a build number close to 300? :)

**Comment by GithubGrisen** [comment]
If you're referring to (r103) I think we're a wee bit off :-)

**Comment by GithubGrisen** [comment]
Just tried r305, and neither send nor recieve works.

**Comment by trickyut** [comment]
I can confirm that the issue is still not solved.
I'm running 16.04(2018-W28) on my OPO and i cannot send/receive MMS.

**Comment by Thieb67k** [comment]
Hello,
It seems to be similar to #12 and #975.
I have the same issue with LGE Nexus 4 (OTA-5 and OTA-6).

**Comment by giiba** [comment]
I just got a new OPP and had this exact issue. MMS messages would just fail and I could not receive them. Then I started digging in apn settings and everything looked fine... but still nothing worked.

Then I noticed a tiny white box on the white background, I touched it and pop up came a little green check mark. Now MMS seems to be working, sending and receiving fine.

I think this bug may just be down to the apn being set and then not activated by default.

See:
![screenshot20190501_063351397](https://user-images.githubusercontent.com/7121141/57019577-ce9e4700-6bdb-11e9-8a2c-b085f7a22394.png)
On a fresh install, that little box was unchecked?

**Comment by Flohack74** [comment]
I was already wondering whats the point of that checkboxes. They might have overdesigned this dialog completly, and now nobody understands how to use that. So yes, it might change smth but my impression was it does not do that. But maybe it works for you :)

**Comment by giiba** [comment]
I cannot say this has been thoroughly tested, but initial results point to my solution being being that funny little box.

**Comment by Flohack74** [comment]
Bummer, seems my carrier does not even offer MMS. I am out for testing I gues...

**Comment by mgresko8** [comment]
Wow, I did not realize there are some checkboxes. But after enabling I
am still not receiving MMS.

Marek

2019-05-02 22:01 GMT+02:00, Florian Leeber <notifications@github.com>:
